src/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -412,6 +412,24 @@ public sealed class FileSystem
         public static void Write(int FileNumber, params object[] Output) { }
         public static void WriteLine(int FileNumber, params object[] Output) { }
     }
+    [Microsoft.VisualBasic.CompilerServices.StandardModuleAttribute]
+    public sealed class Financial
```

Comment:
We can make this a `public static` class and removed the internal constructor

Reply:
Perhaps, but let's keep this as `sealed class` for consistency with other `Module` definitions in the ref assembly.

Reply:
Yes, as @cston said, I followed the the structure of the file to keep consistency.